app/src/main/java/org/andstatus/todoagenda/WidgetInfoStorage.java

```diff
@@ -25,16 +22,12 @@
 /**
  * @author yvolk@yurivolkov.com
  */
-public class CalendarQueryResultsStorage {
+public class WidgetInfoStorage {
```

Comment:
**WidgetInfo**Storage name is much less concrete  than Calendar**QueryResults**Storage thus making purpose of the class harder to understand. This is all about "widget" and "info" here...

Reply:
That's true, but even before, it already stored more than the query results - it stores settings and some device information used for debugging. And now it stores also task results, so while WidgetInfo may be a bad name, I think CalendarQueryResults is also somewhat misleading.

Reply:
No, the class doesn't store settings, there is org.andstatus.todoagenda.prefs.SettingsStorage for that.
It doesn't store device information either: it only adds that information, when we "shareEventsForDebugging"